### PR TITLE
[Android] Updated document for nnstreamer multi-model app

### PR DESCRIPTION
**Pull request by leemgs**
Fixed issue nnstreamer/nnstreamer#2349.

This commit is to update the existing document in order to re-upload
the nnstreamer multi-model application because the application management
of Google Store is changed.

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

---

### CODE REVIEW

**Comment by leemgs** [comment]
@again4you , PTAL

File: api/android/README.md [hunk 1]
```diff
@@ -18,23 +18,23 @@ We assume that you already have experienced Android application developments wit
...
 #
 export JAVA_HOME=/opt/android-studio/jre            # JRE path in Android Studio
-export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (The default path will be "$HOME/Android".)
-#
-# $ANDROID_DEV_ROOT/tools/sdk                    # Android SDK root directory (default location: $HOME/Android/Sdk)
-# $ANDROID_DEV_ROOT/tools/ndk                    # Android NDK root directory (default location: $HOME/Android/Sdk/ndk/<ndk-version>)
-# $ANDROID_DEV_ROOT/gstreamer-1.0                # GStreamer binaries
-# $ANDROID_DEV_ROOT/workspace/nnstreamer         # NNStreamer cloned git repository
-#
+export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (default location: "$HOME/Android".)
```
Comment (dongju-chae) [inline api/android/README.md:31]: myandroid vs android?

File: api/android/README.md [hunk 2]
```diff
@@ -51,17 +51,25 @@ You can see the installation guide [here](https://developer.android.com/studio/i
...
+ * SSL Certificate:  File > Settings ... > Tools > Server Certificates > (x) Accept non-trusted certificates automatically 
+```bash
+./android-studio/bin/studio.sh` file.
 ```
+Finally, install SDK into the `$ANDROID_SDK` folder as follows.
+The `yes` command automatically agrees to the license question for the Android SDK.
+```bash
 $ cd $ANDROID_SDK/tools/bin
 $ yes | ./sdkmanager --licenses
+(Proxy environment: yes | sdkmanager --no_https --proxy=http --proxy_host=10.112.1.184 --proxy_port=8080 --licenses)
```
Comment (jaeyun-jung) [inline api/android/README.md:72]: let's hide host & port num

File: api/android/README.md [hunk 3]
```diff
@@ -51,17 +51,25 @@ You can see the installation guide [here](https://developer.android.com/studio/i
...
 ```
 
-Now, run the android studio from `./android-studio/bin/studio.sh` and install SDK to `$ANDROID_SDK`.
-Next, agree to the licenses for the Android SDK.
+Now, run the **Android Studio** IDE as follows.
+If your network is maintainced a proxy of the office, you need to set-up the proxy  and SSL configuraiton.
+ * Proxy setting: File > Appearance & Behavior > System Settings > HTTP Proxy
+ * SSL Certificate:  File > Settings ... > Tools > Server Certificates > (x) Accept non-trusted certificates automatically 
+```bash
+./android-studio/bin/studio.sh` file.
```
Comment (dongju-chae) [inline api/android/README.md:65]: typo detected

File: api/android/README.md [hunk 4]
```diff
@@ -18,23 +18,23 @@ We assume that you already have experienced Android application developments wit
 First of all, you need to set-up the development environment as following:
 
 ```bash
-$ export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (The default path will be "$HOME/Android".)
+$ export ANDROID_DEV_ROOT=$HOME/myandroid           # Set your own path (default location: $HOME/Android)
```
Comment (dongju-chae) [inline api/android/README.md:21]: I think it's better to use `android`

File: api/android/README.md [hunk 5]
```diff
@@ -18,23 +18,23 @@ We assume that you already have experienced Android application developments wit
...
-export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (The default path will be "$HOME/Android".)
-#
-# $ANDROID_DEV_ROOT/tools/sdk                    # Android SDK root directory (default location: $HOME/Android/Sdk)
-# $ANDROID_DEV_ROOT/tools/ndk                    # Android NDK root directory (default location: $HOME/Android/Sdk/ndk/<ndk-version>)
-# $ANDROID_DEV_ROOT/gstreamer-1.0                # GStreamer binaries
-# $ANDROID_DEV_ROOT/workspace/nnstreamer         # NNStreamer cloned git repository
-#
+export ANDROID_DEV_ROOT=$HOME/android               # Set your own path (default location: "$HOME/Android".)
+
+# $ANDROID_DEV_ROOT/tools/sdk/: Android SDK root directory (default location: $HOME/Android/Sdk)
```
Comment (wooksong) [inline api/android/README.md:33]: Q about (default location: $HOME/Android/Sdk):

Those default locations are the directories which the Android SDK and NDK are installed in by default? I am confusing.

File: api/android/README.md [hunk 6]
```diff
@@ -51,17 +51,25 @@ You can see the installation guide [here](https://developer.android.com/studio/i
 For example,
 ```bash
 $ firefox  https://developer.android.com/studio
```
Comment (wooksong) [inline api/android/README.md:53]: This comment is not for this PR.

```suggestion
$ firefox  https://developer.android.com/studio &
```

**Comment by leemgs** [comment]
This PR will be closed for PR #2378 . :)

**Pull request closed without merging**